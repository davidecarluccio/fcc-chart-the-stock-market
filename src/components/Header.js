import React from 'react';

const Header = () => {
	return (
		<nav className="navbar-fixed">
				<div className="nav-wrapper grey darken-4">
					<a href="/" className="brand-logo">
						<i className="fa fa-line-chart material-icons" aria-hidden="true" />
						Stock Chart
					</a>
				</div>
		</nav>
	);
};

export default Header;
