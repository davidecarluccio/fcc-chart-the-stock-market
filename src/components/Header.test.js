import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
	it('should render a nav component', () => {
		const wrapper = shallow(<Header />);
		expect(wrapper.find('nav')).toHaveLength(1);
	});

	it('should render the whole navbar', () => {
		const tree = renderer
			.create(
				<nav className="navbar-fixed">
					<div className="nav-wrapper grey darken-4">
						<a href="/" className="brand-logo">
							<i
								className="fa fa-line-chart material-icons"
								aria-hidden="true"
							/>
							Stock Chart
						</a>
					</div>
				</nav>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
