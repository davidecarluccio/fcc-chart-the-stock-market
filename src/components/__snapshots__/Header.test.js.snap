// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the whole navbar 1`] = `
<nav
  className="navbar-fixed"
>
  <div
    className="nav-wrapper grey darken-4"
  >
    <a
      className="brand-logo"
      href="/"
    >
      <i
        aria-hidden="true"
        className="fa fa-line-chart material-icons"
      />
      Stock Chart
    </a>
  </div>
</nav>
`;
